docs/entities/registeredservice.md

```diff
@@ -0,0 +1,49 @@
+# RegisteredService
+
+A RegisteredService represents an instance of a running service. This resource can be created by both
+a human operator or a piece of software (agent, bot, script). This flexibility allows for the service provisioning and service discovery operations
+to be decoupled from representation of that service in a service catalog. The service catalog can be consumed by
+Kubernetes applications that are unaware of the lifecycle of the service. The creation of a registered server signals
+and update to the service catalog can be be advertised to the application developers. A RegisteredService can be
```

Comment:
This sentence is not clear to me. What do you mean here?
```
The creation of a registered server signals and update to the service catalog can be be advertised to the application developers.
```

Reply:
changed